.github/workflows/swiftlint.yaml

```diff
@@ -0,0 +1,27 @@
+name: Lint swift
```

Comment:
I really don't know about this new workflow. On one hand, it's consistent with the Kotlin workflow we have. This allows early failure, which goes with the "F" in "F.I.R.S.T.".
But 
1. If you do this, just delete the path detection for Swift in the js-and-android-test workflow. It's not used anywhere else.
2. I don't know if the paths are totally intended so that core is excluded, but there are a few swift files in there too.
3. (These files would either need their own swiftlint configuration, or else move the `.swiftlint.yml` file to `platform`)
4. Using a github action in the marketplace is probably cleaner https://github.com/marketplace/actions/github-action-for-swiftlint We would have to ask @L-E for authorization, though.